### PR TITLE
Droopler phpcs

### PR DESCRIPTION
**Pull request by aliegmann-droptica**
I created GitHub Action for verify Drupal coding standards

### CODE REVIEW

**Comment by grzegorz-pietrzak-droptica** [comment]
@aliegmann-droptica Thanks for adding this, I will put this change into latest develop branch, and include it in the next release

File: .github/workflows/phpcs.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+jobs:
+  Verify:
+    name: "Verify Drupal conding standards"
+
+    runs-on: "ubuntu-latest"
+
+    env:
+      CODE_SNIFFER_PATH: "modules/custom/ themes/custom/"
+      CODE_SNIFFER_IGNORE: "*md,*.css,*.txt,*.info,*interactive_ui/react_slider/js*,*pl-sk/*,*pattern-lab/*,*_twig-components/*"
+      CODE_SNIFFER_EXTENSIONS: "php,inc,install,module,yaml,twig,theme"
```
Comment (mariusz-andrzejewski-droptica) [inline .github/workflows/phpcs.yml:24]: What about the `profile` file extension?
What about the scss, js? Is it also validated?

File: .github/workflows/phpcs.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
```
Comment (mariusz-andrzejewski-droptica) [inline .github/workflows/phpcs.yml:33]: How about settings the version also in the `env` ?

File: .github/workflows/phpcs.yml (hunk 1 above)
Comment (aliegmann-droptica) [reply .github/workflows/phpcs.yml]: @mariusz-andrzejewski-droptica Good point, i added profile extension to check.
About scss and js files, I think better solution would be [Stylelint](https://stylelint.io/) and [ESLint](https://eslint.org/) like in Drupal core

File: .github/workflows/phpcs.yml (hunk 2 above)
Comment (aliegmann-droptica) [reply .github/workflows/phpcs.yml]: Php version moved to env

**Review by mariusz-andrzejewski-droptica — APPROVED** [review]

**Pull request merged**